package.json

```diff
@@ -35,6 +35,8 @@
     "tslint-config-airbnb": "5.11.1",
     "tslint-config-prettier": "1.18.0",
     "tslint-plugin-prettier": "2.0.1",
+    "typedoc": "^0.14.2",
```

Comment:
Do we need this in the root `package.json`?

Reply:
Is the goal to eventually use typedocs throughout the project? If so, it makes sense here, right? If no, I could just move it into the relevant projects.

Reply:
Always makes sense to use it in the relevant projects